### PR TITLE
Configuring linters gitignore

### PR DESCRIPTION
**Pull request by M0rrighan**
Changes made:

Added Welcome page along with simple styling
Added Linters configurations
Added .gitignore file

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: green;
+}
```
Comment (DeliceLydia) [inline style.css:3]: - [ ] Kindly fix linter errors currently they are coming from your stylelint configuration, figure out where they are coming from and fix them

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @M0rrighan 👐, 

Your project is almost done! However, there are some issues that you still need to work on to go to the next project but you are almost there.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] You did well to add a title and a description in your PR but you can get more inspiration from this [Link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) on how to make a professional repo


## Keep Aiming High! 🙌

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (M0rrighan) [reply style.css:3]: Hi @DeliceLydia,
According to the post linked below, if I understood correctly, the same problem was solved before by deleting 'package-lock.json' or by editing the configuration file linters.yml.
https://questions.microverse.org/t/stylelint-error-cannot-find-module-postcss-lib-input/1360/3
Making changes in linters.yml was strongly not recommended and I am not sure if I can just delete the package-lock.json file. What would you recommend?

**Review by mricanho — COMMENTED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @M0rrighan,

**Your project is complete, but, for future projects, wait until a Code Reviewer approves your project to merge the feature branch with the main branch**

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**